### PR TITLE
New package: nagstamon-3.0.2

### PR DESCRIPTION
**Pull request by laulicus**

### CODE REVIEW

File: srcpkgs/nagstamon/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+version=3.0.2
+revision=1
+wrksrc=Nagstamon
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-keyring"
+depends="python3-BeautifulSoup4 python3-dbus python3-keyring python3-lxml python3-psutil python3-PyQt5-multimedia python3-PyQt5-svg python3-requests"
+short_desc="Nagios status monitor for the desktop"
+maintainer="Laszlo Dvornik <laulicus@zoho.com>"
+license="GPL-2"
```
Comment (Johnnynator) [inline srcpkgs/nagstamon/template]: You should use the [SPDX Identifier](https://spdx.org/licenses/) in this case `GPL-2.0-or-later`

File: srcpkgs/nagstamon/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'nagstamon'
+pkgname=nagstamon
+version=3.0.2
+revision=1
+wrksrc=Nagstamon
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-keyring"
+depends="python3-BeautifulSoup4 python3-dbus python3-keyring python3-lxml python3-psutil python3-PyQt5-multimedia python3-PyQt5-svg python3-requests"
```
Comment (Johnnynator) [inline srcpkgs/nagstamon/template]: Pls wrap the line after ~80 chars.

File: srcpkgs/nagstamon/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+short_desc="Nagios status monitor for the desktop"
+maintainer="Laszlo Dvornik <laulicus@zoho.com>"
+license="GPL-2"
+homepage="https://nagstamon.ifw-dresden.de"
+distfiles="https://nagstamon.ifw-dresden.de/files/stable/Nagstamon-${version}.tar.gz"
+checksum=a6e9fbb3c4ad8f88ec8540f62457a7d32b6180e6aa0a5ff28ddacb15bc646f9d
+
+post_install() {
+		chmod 644 ${DESTDIR}/usr/share/pixmaps/nagstamon.svg
```
Comment (maxice8) [inline srcpkgs/nagstamon/template]: only 1 tab of ident

File: srcpkgs/nagstamon/template [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'nagstamon'
+pkgname=nagstamon
+version=3.0.2
+revision=1
+wrksrc=Nagstamon
+build_style=python3-module
```
Comment (maxice8) [inline srcpkgs/nagstamon/template]: please add pycompile_module=

File: srcpkgs/nagstamon/template [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'nagstamon'
+pkgname=nagstamon
+version=3.0.2
+revision=1
```
Comment (Johnnynator) [inline srcpkgs/nagstamon/template]: you can add `noarch=yes` after this line

File: srcpkgs/nagstamon/template (hunk 1 above)
Comment (laulicus) [reply srcpkgs/nagstamon/template]: In the source tarball, I find this is only GPL2. GPL-2.0-only would be OK?

File: srcpkgs/nagstamon/template (hunk 2 above)
Comment (laulicus) [reply srcpkgs/nagstamon/template]: Fixed.

File: srcpkgs/nagstamon/template (hunk 3 above)
Comment (laulicus) [reply srcpkgs/nagstamon/template]: Fixed.

File: srcpkgs/nagstamon/template (hunk 4 above)
Comment (laulicus) [reply srcpkgs/nagstamon/template]: Fixed

File: srcpkgs/nagstamon/template (hunk 5 above)
Comment (laulicus) [reply srcpkgs/nagstamon/template]: Fixed

File: srcpkgs/nagstamon/template (hunk 1 above)
Comment (Johnnynator) [reply srcpkgs/nagstamon/template]: Take a look at the headers of the source files. e.g. https://github.com/HenriWahl/Nagstamon/blob/master/Nagstamon/__init__.py

Comment (laulicus) [reply srcpkgs/nagstamon/template]: Thank you. Fixed.

**Comment by laulicus** [comment]
Please review my new commit, I had tried to fix every problems you wrote.

**Pull request closed without merging**